.github/ISSUE_TEMPLATE/release-upgrade-testing.md

```diff
@@ -0,0 +1,159 @@
+---
+name: Release Demo and Upgrade Testing
+about: |
+  This template can be used to track the upgrade testing of demos from the outgoing stable to
+  the new release, and new release from scratch before officially marking the new release as stable.
+title: "chore(tracking): Test demos on nightly versions for YY.M.X"
+labels: ['epic']
+assignees: ''
+---
+
+<!--
+    Make sure to update the link in '.github/ISSUE_TEMPLATE/release.md' when
+    you change the filename.
+-->
```

Comment:
note: This needs to be fixed in this file and in the `pre-release-upgrade-testing.md` template.

```suggestion
<!--
    Make sure to update the link in 'stackabletech/issues/.github/ISSUE_TEMPLATE/release.md' when
    you change the filename.
-->
```